tests/v2/actions/kubeapi/nodes/nodes.go

```diff
@@ -0,0 +1,47 @@
+package nodes
+
+import (
+	"context"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/pkg/api/scheme"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// NodeGroupVersionResource is the required Group Version Resource for accessing nodes in a cluster,
+// using the dynamic client.
+var NodeGroupVersionResource = schema.GroupVersionResource{
+	Group:    "",
+	Version:  "v1",
+	Resource: "nodes",
+}
+
+// GetNodes returns nodes with metav1.TypeMeta, metav1.ObjectMeta, NodeSpec, and NodeStatus to be used to gather more information from nodes
+func GetNodes(client *rancher.Client, clusterID string, listOpts metav1.ListOptions) ([]corev1.Node, error) {
```

Comment:
I feel like a lot of the kubeapi stuff could stay in extensions. I don't understand how they wouldn't qualify.

Reply:
Same here, this is just doing a simple CRUD that is converting. We should really be using the controller to do this anyways. This fits this point
should not only directly convert -> this would make it an action